### ISSUE TITLE
Add healthcheck and environment controllers and use cases

### ISSUE THREAD

**Issue by acostapazo** [alice-biometrics/petisco#23]
It'd be nice have available some pre-configured controllers to delegate healthcheck and environment entry points.

Then, we could refer it with:

e.g swagger.yml
```yml
paths:
  /healthcheck:
    get:
      summary: Returns healthcheck information.
      operationId: petisco.healthcheck
      responses:
        200:
          description: The healthcheck was successful.
          content:
            application/json:
              schema:
                type: string
        503:
          description: Error connecting with configured persistence (Database Connection)

  /environment:
    get:
      summary: Returns environment information.
      operationId: petisco.environment
      security:
        - ApiKeyAuth: []
      responses:
        200:
          description: The information has been returned.
          content:
            application/json:
              schema:
                type: string
        401:
          description: Access token or API key is missing or invalid.
        500:
          description: Unexpected Error obtaining environment info
```

**Comment by acostapazo** [comment]
Solved in https://github.com/alice-biometrics/petisco/commit/fa1dc24af2dabb86220e103c9c1f013a666cebb7

**Comment by acostapazo** [comment]
Available in https://github.com/alice-biometrics/petisco/releases/tag/v0.19.2
